packages/peregrine/lib/talons/AuthModal/useAuthModal.js

```diff
@@ -49,13 +49,9 @@ export const useAuthModal = props => {
         closeDrawer();
     }, [closeDrawer, showMainMenu]);
 
-    const handleCreateAccount = useCallback(
-        async values => {
-            await createAccount(values);
-            showMyAccount();
-        },
-        [createAccount, showMyAccount]
-    );
+    const handleCreateAccount = useCallback(() => {
+        showMyAccount();
```

Comment:
See how this callback no longer cares about the `createAccount` action. So clean!